bin/lib/homepage.js

```diff
@@ -0,0 +1,126 @@
+require('dotenv').config();
```

Comment:
can we tweak the dotenv code so it doesn't report an error when running on Heroku with no .env file?